client/src/JobPaginatorZoom.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import PaginatorPreview from "./PaginatorPreview";
+import PaginatorZoomy from "./PaginatorZoomy";
+
+const JobPaginatorZoom = ({ zoom, getImageUrl }) => {
```

Comment:
This might need a better name, since it's only a Zoom when zoom is set to true. Maybe JobPaginatorZoomToggle?

Reply:
Or JobPaginatorTogglableZoom?

Reply:
Making change.